src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java

```diff
@@ -0,0 +1,48 @@
+package co.novu.api.workflowoverrides;
+
+import co.novu.api.workflowoverrides.pojos.WorkflowOverride;
+
+import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
+import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.PUT;
+import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
+
+import java.util.Map;
+
+public interface WorkflowOverrideApi {
+
+    String ENDPOINT = "workflow-overrides";
+
+    @POST(ENDPOINT)
+    Call<WorkflowOverride> createWorkflowOverride(@Body CreateWorkflowOverrideRequest request);
```

Comment:
This response body still doesn't conform with what is in the doc. Pls consider using https://www.jsonschema2pojo.org/